appinventor/appengine/src/com/google/appinventor/client/DesignToolbar.java

```diff
@@ -112,11 +112,13 @@ public void setCurrentScreen(String name) {
   }
 
   private static final String WIDGET_NAME_TUTORIAL_TOGGLE = "TutorialToggle";
+  private static final String WIDGET_NAME_HELP_TOGGLE = "HelpToggle";
   private static final String WIDGET_NAME_ADDFORM = "AddForm";
   private static final String WIDGET_NAME_REMOVEFORM = "RemoveForm";
   private static final String WIDGET_NAME_SCREENS_DROPDOWN = "ScreensDropdown";
   private static final String WIDGET_NAME_SWITCH_TO_BLOCKS_EDITOR = "SwitchToBlocksEditor";
   private static final String WIDGET_NAME_SWITCH_TO_FORM_EDITOR = "SwitchToFormEditor";
+  private static final String helpURL ="http://localhost:8888/reference/components/userinterface.html";
```

Comment:
Don't hardcode the `localhost:8888` reference. Instead, using a relative URL starting with `/`